contracts/DAOFunding.sol

```diff
@@ -0,0 +1,18 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+// TODO(system): implement logic around DAO (deposit, withdraw, distribute)
+contract DAOFunding is Initializable {
```

Comment:
Suggestion, lets rename using the definitions provided by the prompt:
* DAOFunding -> Investment
* children -> investments